index.html

```diff
@@ -5,8 +5,44 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Henry Cadena Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Henry Cadena Portfolio Mobile First</h1>
+    <header class="container-header">
+      <a href="#">
+        <h3 class="logo">Henry Cadena</h3>
+      </a>
```

Comment:
- [ ] Good job in wraping your logo with an anchor tag👍🏽 But please remove your **_h3_** tag that is nested in your anchor tag **_a_** 🙏🏽 Because it is generally not considered good practice to wrap a heading  (**_h1-h6_**) inside an anchor **_(a)_** tag in HTML. 😄 😄 The reason is that headings are **structural elements** used to define the hierarchy and organization of content on a webpage, while anchor tags are primarily used to create **hyperlinks**.
You can change your code to this instead 👇🏽 👇🏽 

```sh
<a href="#" class="logo">Henry Cadena</a>
```